README.md

```diff
@@ -92,6 +92,13 @@ Layouts are defined using the [define_layout!](https://docs.rs/binary-layout/lat
 2. The [FieldView](https://docs.rs/binary-layout/latest/binary_layout/struct.FieldView.html) API that wraps a slice of storage data and remembers it in a `View` object, allowing access to the fields without having to pass in the packed data slice each time. This is the API used in the example above. See [FieldView](https://docs.rs/binary-layout/latest/binary_layout/struct.FieldView.html) for another example.
 
 ### Supported field types
+
```

Comment:
let's move this to the end of the list. Since people likely are more interested in the primitive integer types than in the ZSTs, they should see those first in the documentation.